diff -uprNbd pavucontrol-2.0.orig/src/mainwindow.cc pavucontrol-2.0/src/mainwindow.cc
--- pavucontrol-2.0.orig/src/mainwindow.cc	2012-12-17 11:19:04.000000000 +0200
+++ pavucontrol-2.0/src/mainwindow.cc	2015-03-25 18:47:45.616433552 +0200
@@ -481,7 +481,7 @@ static void suspended_callback(pa_stream
 static void read_callback(pa_stream *s, size_t length, void *userdata) {
     MainWindow *w = static_cast<MainWindow*>(userdata);
     const void *data;
-    double v;
+    double v = 0;
 
     if (pa_stream_peek(s, &data, &length) < 0) {
         show_error(_("Failed to read data from stream"));
@@ -491,6 +491,7 @@ static void read_callback(pa_stream *s,
     assert(length > 0);
     assert(length % sizeof(float) == 0);
 
+    if (data)
     v = ((const float*) data)[length / sizeof(float) -1];
 
     pa_stream_drop(s);
